src/app/shared/services/data.service.ts

```diff
@@ -74,4 +74,18 @@ export class DataService {
   getRecentItemsForSale(): Item[] {
     return this.dummyDataService.getRecentItemsForSale();
   }
+
+  addNewItem(
+    itemName: string,
```

Comment:
This is just a design choice, but you can choose to pass in an object instead of all of these attributes.

It's more useful because then you don't have to worry about what specific argument location they are in. Also, you can dereference those attributes from the objects directly:

For example,

```
addNewItem({
  itemName,
  sellerID,
  conditionTypeID,
  categoryTypeID,
  description,
  price,
  isItemSold,
  rating,
  pictureURL}) {
  ...
}
```